tests/integration/kubernetes/setup.sh

```diff
@@ -16,6 +16,13 @@ set_runtime_class() {
     sed -i -e "s|runtimeClassName: kata|runtimeClassName: kata-${KATA_HYPERVISOR}|" runtimeclass_workloads/*.yaml
 }
 
+set_hypervisor_path() {
+    if [[ "${KATA_HOST_OS}" = "cbl-mariner" ]]; then
+        mariner_clh_path="/usr/bin/cloud-hypervisor"
+        find runtimeclass_workloads/*.yaml -exec yq write -i {} 'metadata.annotations[io.katacontainers.config.hypervisor.path]' "${mariner_clh_path}" \;
```

Comment:
Actually, to address this and other comments, how about in `kata-deploy.sh` we configure Kata to 1) set `valid_hypervisor_paths = ["/*"]` and 2) set `enable_annotations = ".*"`?

Reply:
I've been going back and forth on this one.

Ideally we could have `/usr/bin/cloud-hypervisor` added as part of the `valid_hypervisor_path`, enable the annotation, and set it accordingly.

One thing that makes me worried here is if we have different versions of cloud-hypervisor.
Let's say the one we provide is 32.0, while the one supported on Mariner is 27.0.  There may be API changes in the runtime code that will simply make 27.0 not work.

cc @likebreath for his opinion on this one.

Reply:
> One thing that makes me worried here is if we have different versions of cloud-hypervisor.
Let's say the one we provide is 32.0, while the one supported on Mariner is 27.0. There may be API changes in the runtime code that will simply make 27.0 not work.

Cc @liuw - this PR deploys Mariner's CLH during CI instead of using the upstream version. Thoughts?

Reply:
If you want to track upstream Kata then you should also track upstream Cloud Hypervisor.